chronos.nimble

```diff
@@ -32,10 +32,11 @@ proc run(args, path: string) =
 
 task test, "Run all tests":
   for args in [
-      "-d:useSysAssert -d:useGcAssert",
-      "-d:chronosStackTrace -d:chronosStrictException",
+      "-d:debug",
+      "-d:debug -d:chronosv4",
       "-d:release",
-      "-d:release -d:chronosFutureTracking",
+      "-d:release -d:chronosv4",
+      "-d:release -d:useSysAssert -d:useGcAssert",
```

Comment:
why `chronosFutureTracking` was removed?

Reply:
it's now enabled by `chronosPreviewV4` (previously `chronosv4`)

Reply:
But it should not be enabled with by default.

Reply:
ah, sorry, I misread which feature you meant - so if you look at `config.nim`, the future tracking would now be enabled by default `-d:debug` - you can also separately enable or disable it with `-d:chronosFutureTracking=true|false` which overrides `debug` - is this too aggressive?